frontend/src/app/settings/page.tsx

```diff
@@ -1,9 +1,36 @@
+"use client";
 import React from "react";
+import { useGetUser } from "@/hooks/useGetUser";
 
 const Settings = () => {
+  const { data, error, isLoading } = useGetUser();
```

Comment:
ここは好みですが自分は
```suggestion
  const { data: userData, error, isLoading } = useGetUser();
```
のように何のデータなのかがわかるように別名をつけるようにしてます！

Reply:
確かにその方がずっとわかりやすいです！ありがとうございます。